k8s/namespaces/rd/rd-user-profile-api-integration/rd-user-profile-api-integration.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/user-profile-api:prod-ba55cec-20210831142043
+      image: hmctspublic.azurecr.io/rd/user-profile-api:prod-ba55cec-20210831142043   #{"$imagepolicy": "flux-system:rd-user-profile-api-integration"}
```

Comment:
Can you change this policy to rd-user-profile-api? It looks to be using the same, I would remove the image repo/policy for `rd-user-profile-api-integration`

Reply:
Image policy/repo now updated.